sidebars.js

```diff
@@ -182,6 +182,11 @@ const sidebars = {
               type: "doc",
               label: "Helpful CLI commands",
               id: "nodes/celestia-app-commands"
+            },
+            {
+              type: "link",
+              label: "Specifications",
+              href: "https://celestiaorg.github.io/celestia-app/",
```

Comment:
[not blocking] my personal opinion is that the celestia-app specs don't need to be linked in the sidebar twice. I think the link below under **Specifications** seems preferable so proposal to remove this one.

Reply:
understood, thanks for the feedback. since the "specifications" is on the "Concepts" category, do you think that is still preferable to the "Run a Node" category where the above is currently in the PR?